packages/qgrid-ngx-plugins/src/lib/action-bar/action-bar.component.ts

```diff
@@ -36,4 +50,14 @@ export class ActionBarComponent implements OnInit {
 		const { model } = this.plugin;
 		return model.action().items;
 	}
+
+	isSorted(array: Action[]): boolean {
+		for (let i = 0; i < array.length - 1; i++) {
+			if (array[i].command.priority > array[i + 1].command.priority) {
```

Comment:
will be a little bit more readable if do:
const action = actions[i];
const nextAction = actions[i + 1];